Mathlib/Algebra/Polynomial/AlgebraMap.lean

```diff
@@ -177,6 +186,19 @@ theorem mapAlgHom_eq_eval₂AlgHom'_CAlgHom (f : A →ₐ[R] B) : mapAlgHom f =
     (CAlgHom.comp f) X (fun a => (commute_X (C (f a))).symm) := by
   rfl
 
+lemma coeff_mapAlgHom_apply (f : A →ₐ[R] B) (p : A[X]) (n : ℕ) :
+    coeff (mapAlgHom f p) n = f (coeff p n) := by
+  simp
+
+lemma lcoeff_comp_mapAlgHom_eq (f : A →ₐ[R] B) (n : ℕ) :
+    LinearMap.comp (AlgHom.toLinearMap f) ((lcoeff A n).restrictScalars R) =
+      LinearMap.comp ((lcoeff B n).restrictScalars R) (mapAlgHom f).toLinearMap := by
```

Comment:
```suggestion
    f.toLinearMap ∘ₗ (lcoeff A n).restrictScalars R =
      (lcoeff B n).restrictScalars R ∘ₗ (mapAlgHom f).toLinearMap := by
```
(one can also omit the `restrictScalars R` because coercion inserts it for you but I am less sure if we want to rely on this)
And the equality should probably flipped not least because of the lemma name.